chromadb/test/ef/test_spacy_ef.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from chromadb.utils.embedding_functions import SpacyEmbeddingFunction
+
+input_list = ["great work by the guy", "Super man is that guy"]
+model_name = "en_core_web_md"
+unknown_model = "unknown_model"
+
+
+def test_spacyembeddingfunction_isnotnone_wheninputisnotnone():
+    spacy_emb_fn = SpacyEmbeddingFunction(model_name)
+    assert spacy_emb_fn(input_list) is not None
```

Comment:
can we also check that this a list of lists of floats

Reply:
Resolved